capabilities/navigation/nearby-poi/profile.supr

```diff
@@ -0,0 +1,65 @@
+"""
+Find nearby points of interest.
+
+Find points of interest near the given location using map services.
+"""
+name = "navigation/nearby-poi"
+version = "1.0.0"
+
+"""
+Find nearby points of interest
+
+Find points of interest near the given location.
+"""
+usecase NearbyPoi {
+	input {
+		"""
+		center
+		Center of the search
+		"""
+		center! Coordinates!
+		
+		"""
+		radius
+		Radius of the search
+		"""
+		radius! number!
+		
+		"""
+		categories
+		Optional categories filter
+		Points belonging to at least one of these categories are returned
+		"""
+		categories [InterestCategory!]
+	}
+
+	result [
+		{
+			"Coordinates of this point"
+			coordinates! Coordinates!
+			"Name of the point of interest"
+			name! string!
+			"Categories this point belongs to"
+			categories! [InterestCategory!]
+		}
+	]
+
+	error {
+		"Human-readable status description"
+		status! string!
+		"Human-readable error message"
+		message! string!
+	}
+}
+
+model Coordinates {
+	latitude! number!
+	longitude! number!
+}
+
+model InterestCategory enum {
+	RESTAURANT,
+	CAFE,
+	BAR
+	// TODO: More categories
```

Comment:
Can you please add some more categories from [here](https://developers.google.com/maps/documentation/places/web-service/supported_types) before we publish this? Probably most generic ones to be able to work with overpass as well.